schema/component.schema.json

```diff
@@ -46,6 +46,57 @@
               },
               "_backboneForms": "TextArea"
             },
+            "_imageAlignment": {
+              "type": "string",
+              "title": "Image alignment",
+              "description": "Defines the alignment of the pop up image. Left: Image aligned to the left of the text area. Top: Image aligned above the text area. Right: Image aligned to the right of the text area. Bottom: Image aligned below the text area. The default alignment is `right`.",
+              "default": "right",
+              "enum": [
+                "left",
+                "top",
+                "right",
+                "bottom"
+              ],
+              "_backboneForms": "Select"
+            },
+            "_graphic": {
+              "type": "object",
+              "title": "Graphic",
+              "default": {},
+              "required": [
+                "_src"
```

Comment:
Same comment as before but for new schema

Reply:
Amended in 483c09b0d9f332cec391c1972ace7275c20e08b5